src/components/attendee/AttendeeLayout.tsx

```diff
@@ -26,10 +31,16 @@ import styles from "./AttendeeLayout.module.scss";
 
 interface _AttendeeLayoutProps {
   userId: string;
+  space: SpaceWithId;
 }
 
-const _AttendeeLayout: React.FC<_AttendeeLayoutProps> = ({ userId }) => {
+const _AttendeeLayout: React.FC<_AttendeeLayoutProps> = ({ userId, space }) => {
```

Comment:
I'm not sure if entire `space` is the way to go here. If we just pass the banner URL (keep props down to simple IDs or paths), there will be less coupling between entire space and the layout. This might provide for more flexibility down the road